src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/gateways/PrisonApiGateway.kt

```diff
@@ -3,24 +3,24 @@ package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.gateways
 import org.springframework.core.ParameterizedTypeReference
 import org.springframework.stereotype.Service
 import org.springframework.web.reactive.function.client.WebClient
+import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.entities.RestrictedPatient
 import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.request.CreateExternalMovement
-import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.request.DischargeToHospitalRequest
 import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.response.Agency
 import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.response.OffenderBookingResponse
 
 @Service
 class PrisonApiGateway(private val prisonApiClientCreds: WebClient) {
-  fun dischargeToHospital(dischargeToHospitalDetails: DischargeToHospitalRequest) =
+  fun dischargeToHospital(newRestrictedPatient: RestrictedPatient) =
```

Comment:
Not sure about the new name - this data does not represent the patient themselves, rather the movement of the patient. IMO the original name (which wasn't made up by me) is more representative.

Reply:
this is the exsiting name.  There were two parameters and now there are one.